README.md

```diff
@@ -20,3 +20,43 @@ The plot with Shuxin's list generated from Lidia's and Taylor's papers:
 The plot with CCW's list but screening for all 25 codes:
 
 [ADRDtrend.pdf](https://github.com/ShuxinD/ADRDdata/files/8043033/ADRDtrend.pdf)
+
+## Data Dictionary 
+
+| Fieldname | Source |  Description | Role |
+| --------- | ------ | ------------ | ---- |
+| QID | Medicare | Person's ID | Cohort |
```

Comment:
I think the role of QID can be just "ID", "Cohort" is not accurate for this I think